verifier/credentialsConfig.go

```diff
@@ -68,24 +62,47 @@ func InitServiceBackedCredentialsConfig(repoConfig *config.ConfigRepo) (credenti
 func (cc ServiceBackedCredentialsConfig) fillStaticValues() {
 	for _, configuredService := range cc.initialConfig.Services {
 		logging.Log().Debugf("Add to service cache: %s", configuredService.Id)
-		cc.serviceCache.Add(configuredService.Id, configuredService, cache.NoExpiration)
+		err := common.GlobalCache.ServiceCache.Add(configuredService.Id, configuredService, cache.NoExpiration)
+		if err != nil {
+			logging.Log().Errorf("failed caching configured service in fillStaticValues(): %v", err)
```

Comment:
If adding the configured values fails, the error should be escalated up to killing the program imho